src/tools/genpolicy/rules.rego

```diff
@@ -785,11 +785,15 @@ allow_var(p_process, i_process, i_var, s_name, s_namespace) {
 
     p_name_value[0] == name_value[0]
 
-    # TODO: should these be handled in a different way?
     always_allowed = ["$(resource-field)", "$(todo-annotation)"]
```

Comment:
if we go with this route, we no longer need resource-field, todo-annotation distinction. These could all be $(validate-from-settings)